jenkins/map.jinja

```diff
@@ -11,6 +11,7 @@
             'name': 'Your humble Jenkins',
         },
         'optional_groups': [],
+        'repo_location': "",
```

Comment:
Your SLS looks at `jenkins.jobs.repo_location` but you are creating `jenkins.repo_location`